src/sentry/static/sentry/app/routes.jsx

```diff
@@ -370,6 +370,7 @@ function routes() {
             component={errorHandler(OrganizationContext)}
           >
             <Route component={errorHandler(OrganizationSettingsLayout)}>
+              {hooksOrgRoutes}
```

Comment:
I removed this temporarily because some routes in getsentry weren't updated yet (i.e. usage and billing).

Reply:
oh, that makes sense. i'll drop the line